pkg/kube/cluster-init.sh

```diff
@@ -734,6 +715,35 @@ check_cluster_config_change() {
     logmsg "Check cluster config change done"
 }
 
+# save the /var/lib to /persist/kubelog/save-var-lib
+save_var_lib() {
+  local dest_dir="/persist/kubelog/save-var-lib"
+  # Check if destination directory exists, if not create it
+  if [ ! -d "$dest_dir" ]; then
+    mkdir -p "$dest_dir"
+  fi
+
+  # Remove everything in the destination directory
+  rm -rf "${dest_dir:?}"/*
+
+  # Copy all contents from /var/lib to destination directory
+  cp -a /var/lib/. "$dest_dir"
+}
+
+# Function to restore contents from /persist/kube/log/save-var-lib back to /var/lib
+restore_var_lib() {
+  local source_dir="/persist/kube/log/save-var-lib"
```

Comment:
@naiming-zededa this source_dir doesn't match dest_dir above in save_var_lib()

Reply:
will correct that.